java/flight/src/main/java/org/apache/arrow/flight/FlightClient.java

```diff
@@ -69,7 +69,7 @@
    */
   public FlightClient(BufferAllocator incomingAllocator, Location location) {
     final ManagedChannelBuilder<?> channelBuilder = ManagedChannelBuilder.forAddress(location.getHost(),
-        location.getPort()).maxTraceEvents(0).usePlaintext();
+        location.getPort()).maxTraceEvents(0).maxInboundMessageSize(Integer.MAX_VALUE).usePlaintext();
```

Comment:
move both the constants to a common place, would be better i guess..
also do we it would be useful for the user to be able to set a custom value (through a system property?)

Reply:
I'll move the constants.

It might be useful, but so far we've wanted not to leak gRPC details to the user. And if we were to expose them somehow, I'd rather we provide a way to directly build a client on top of a gRPC Channel instead of having to wrap every property that might need changing.